docs/development/windows_support.md

```diff
@@ -130,19 +134,22 @@ You will need:
   editor like VSCode, CMake can discover the compiler among other "kits". If you
   use the command line, see
   https://learn.microsoft.com/en-us/cpp/build/building-on-the-command-line?view=msvc-170.
-  (typically run the appropriate `vcvarsall.bat`)
+  (typically run the appropriate `vcvarsall.bat`, or `Start > Developer Command Prompt for VS 2022` )
 
 > [!TIP]
 > Some of these tools are available via package managers like
 > https://github.com/chocolatey/choco
 >
+> As noted above, this will automatically configure necessary Visual Studio componenents, Unix tools, and cmake versions for a successful build.
+>
 > ```
-> choco install git
-> choco install cmake
-> choco install ninja
-> choco install ccache
-> choco install sccache
-> choco install python
+> choco install visualstudio2022buildtools -y --params "--add Microsoft.VisualStudio.Component.VC.Tools.x86.x64 --add Microsoft.VisualStudio.Component.VC.CMake.Project --add Microsoft.VisualStudio.Component.VC.ATL --add Microsoft.VisualStudio.Component.Windows11SDK.22621"
+> choco install git.install -y --params "'/GitAndUnixToolsOnPath'"
+> choco install cmake --version=3.31.0 -y
```

Comment:
We could also provide a python requirements file or just instructions for tools that are installable via pip:

* https://pypi.org/project/cmake/
* https://pypi.org/project/ninja/

Then developers can install more tools into a venv instead of system wide if they want.